beps/0004-scaffolder-task-idempotency/README.md

```diff
@@ -82,12 +83,11 @@ We believe that idempotency is the best way to do it. Idempotency allows to reru
 
 ### Serialization of workspace
 
-We believe that a serialization of workspace is a way to achieve re-running the task on a non-sticky way.
-That means that the task can be restored and retried on a different scaffolder node.
+We believe that a serialization of workspace is a way to achieve re-running the task on a non-sticky way. This means that the task can be restored and retried on a different scaffolder backend instance or node. This serialization can be stored in the database, or perhaps additional modules could be installed to provide additional options for storing this serialized workspace data.
```

Comment:
```suggestion
We believe that serialization of workspaces is the way to achieve re-running the task in a non-sticky way. This means that the task can be restored and retried on a different scaffolder task worker. This serialization can be stored in the database, or perhaps additional modules could be installed to provide additional options for storing this serialized workspace data since it may be large in some cases.
```